sdk/examples/CreateFileExample.cc

```diff
@@ -0,0 +1,59 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include "Client.h"
+#include "ED25519PrivateKey.h"
+#include "FileCreateTransaction.h"
+#include "FileId.h"
+#include "PublicKey.h"
+#include "TransactionReceipt.h"
+#include "TransactionResponse.h"
+
+#include <iostream>
+#include <string>
+
+using namespace Hedera;
+
+int main(int argc, char** argv)
+{
+  if (argc < 3)
+  {
+    std::cout << "Please input account ID and private key" << std::endl;
+    return 1;
+  }
+
+  // Get a client for the Hedera testnet, and set the operator account ID and key such that all generated transactions
+  // will be paid for by this account and be signed by this key.
+  Client client = Client::forTestnet();
+  client.setOperator(AccountId::fromString(argv[1]), ED25519PrivateKey::fromString(argv[2]));
+
+  // The file is required to be a byte array, you can easily use the bytes of a file instead.
+  std::string fileContents = "Hedera hashgraph is great!";
+
+  const FileId fileId = FileCreateTransaction()
+                          .setKeys({ client.getOperatorPublicKey() })
+                          .setContents(fileContents)
```

Comment:
Should there be some kind of ```fileContent``` validation? When ```setContents``` is called, the ```fileContent``` is not validated.